docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,102 @@ first number
+1. When using emojis, omit the `::` around the icon name (e.g., use `{icon="emoji"}` instead of `{icon=":emoji:"}`).
+1. If an item has a specified icon, that icon will be used for it and for subsequent items at that level. 
+1. If any item at a level is customized, the first item at that level must also be customized. If not, the list will be invalidated.
+</box>
+
+**The icon's appearance can be further customized by adding a `class` attribute.**
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 <br>
+  Item 1 line 2  {icon="fas-file-code" class="text-warning"}
+* Item 2 
+  Item 2 line 1 continue 
+  * Item 2.1 {icon="fas-code-branch" class="text-success"}
+  * Item 2.2 {class="text-white"}
+* Item 3
+  * Item 3 {icon="/images/deer.jpg" width="30px" height="17px" class="rounded text-white"}
+</variable>
+</include>
+
+<box type="definition" seamless>
+Each item level attribute supersedes the default one. For instance, the class attribute of item 2.2 overrides the default attribute of item 2.1.
+</box>
+
+</div>
+
+**You can adjust the icon's size by using the `size` attribute.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+
+* Item 1 {icon="/images/deer.jpg" width="30px" height="17px"}
+* Item 2 {icon="/images/deer.jpg" width="60px" height="34px"}
+* Item 3 {icon="/images/deer.jpg" width="90px" height="51px"}
+* Item 4 {icon="/images/deer.jpg" width="120px" height="68px"}
+* Item 5 {icon="/images/deer.jpg" width="150px" height="84px"}
+
+</variable>
+</include>
+
+<box type="definition" seamless>
+
+1. All CSS units are acceptable for size. 
+2. If height is omitted, width will be used instead, and vice versa. If both are omitted, the size defaults to the smallest icon size.
+</box>
+
+**You can apply Markdown's heading and paragraph syntax within the list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+
+* ## Heading 1: MarkBind Overview {icon="/images/list.png" width="65px" class="rounded"}
+  
+  Content 1: MarkBind is a tool for generating dynamic, text-heavy websites from Markdown. It supports various syntax schemes for enhanced content dynamism.
+
+* ## Heading 2 Features {icon="/images/toolbox.png" width="65px" class="rounded"}
+  
+  Content 2: MarkBind offers built-in features like icons, emoji, and search functionality. It also supports content reuse and multiple organization methods.
+
+* ## Heading 3 Setup and Integration {icon="/images/setup.png" width="65px" class="rounded"}
+  
+  Content 3: MarkBind is easy to install, modify, and deploy. It integrates seamlessly with software project workflows and offers a live preview feature.
```

Comment:
```suggestion
* ## Heading 1: MarkBind Overview {icon="/images/list.png" width="65px" class="rounded"}
  
  Content 1: MarkBind is a tool for generating dynamic, text-heavy websites from Markdown.
  It supports various syntax schemes for enhanced content dynamism.

* ## Heading 2 Features {icon="/images/toolbox.png" width="65px" class="rounded"}
  
  Content 2: MarkBind offers built-in features like icons, emoji, and search functionality.
  It also supports content reuse and multiple organization methods.

* ## Heading 3 Setup and Integration {icon="/images/setup.png" width="65px" class="rounded"}
  
  Content 3: MarkBind is easy to install, modify, and deploy.
  It integrates seamlessly with software project workflows and offers a live preview feature.
```

Added line breaks to avoid horizontal scrolling in the rendered page.

Reply:
Also, don't go 'meta' with examples. Doing so can make them harder to understand for the unsuspecting reader. For example, the content of this example should not be related to MarkBind at all, so that it is clear they are sample/dummy content and not actual UG content.